PIL/Image.py

```diff
@@ -504,26 +535,30 @@ class Image(object):
     _close_exclusive_fp_after_loading = True
 
     def __init__(self):
+        # type: () -> None
         # FIXME: take "new" parameters / other image?
         # FIXME: turn mode and size into delegating properties?
         self.im = None
         self.mode = ""
         self.size = (0, 0)
-        self.palette = None
-        self.info = {}
+        self.palette = None  # type: Optional[ImagePalette.ImagePalette]
+        self.info = {}  # type: Dict[Text, Any]
```

Comment:
Right, so leave it at `Any` for now? That said, I think I read that it could be better to use `object` where it could be 'anything', but is that going to work for all the types in the python2 that PIL[low] supports?